evenOccurences/evenOccurences.js

```diff
@@ -11,5 +11,21 @@
  *    */
 
 const evenOccurence = (arr) => {
-  // Your code here.
+  let newObj = {};
```

Comment:
Nicely done. I commonly use objects for algorithms like this to hold placeholders in memory.